src/app/dashboard/dashboard.test.js

```diff
@@ -45,13 +44,8 @@ describe('Dashboard', () => {
             expect(sut.isDashboardEmpty()).toBe(true);
         });
 
-        it('should NOT show empty dashboard if there are sensors', () => {
-            sut.sensors = [{}];
-            expect(sut.isDashboardEmpty()).toBe(false);
-        });
-
-        it('should NOT show empty dashboard if there are executors', () => {
-            sut.sensors = [{ executor: true }];
+        it('should NOT show empty dashboard if there is at least one device', () => {
```

Comment:
the test is checking wether dashboard is visible but the expectation is checking wether dashboard is empty. It would be good to correct one or another.